docs/src/main/asciidoc/security-openid-connect.adoc

```diff
@@ -212,10 +212,10 @@ Open a xref:dev-ui.adoc[Dev UI] available at http://localhost:8080/q/dev[/q/dev]
 
 You will be asked to login into a `Single Page Application` provided by `OpenID Connect Dev UI`:
 
- * Login as `alice` (password: `alice`) who has a `user` role
+ * Login as `bob` (password: `bob`) who has a `user` role
  ** accessing `/api/admin` will return `403`
  ** accessing `/api/users/me` will return `200`
- * Logout and login as `admin` (password: `admin`) who has both `admin` and `user` roles
+ * Logout and login as `alice` (password: `alice`) who has both `admin` and `user` roles
```

Comment:
Looks like you do not have a realm imported by `DevServices for Keycloak`.

Can you clarify which Quarkus version do you work with ?

Reply:
@aona42 Or, if you are testing Keycloak launched from a command line with a docker then make sure a realm is imported, it is mentioned in the guide

Reply:
I am using the latest 2.8.2.Final. When I use the mentioned standalone keycloak with the linked realm, then there is admin+alice. When I use the dev services (which I assume is the intention here, because the dev console is mentioned a few lines above) then there is only alice+bob.

Reply:
Linked realm for standalone: https://github.com/quarkusio/quarkus-quickstarts/tree/main/security-openid-connect-quickstart/config/quarkus-realm.json

Dev services: https://quarkus.io/guides/security-openid-connect-dev-services#keycloak-initialization

Reply:
Ok I think I know what the problem is: I haven't downloaded the linked realm and put it on the classpath. Then the dev services default realm is used, which differs from the linked one.

Reply:
I have added another sentence to the dev services section, trying to clarify things.